### ISSUE TITLE
correspondence between pertained models and tables

### ISSUE THREAD

**Issue by linlililin123456** [zhoudaquan/rethinking_bottleneck_design#7]
@zhoudaquan Hi thanks for sharing the work, as from https://github.com/zhoudaquan/rethinking_bottleneck_design/issues/6 could you pls give a correspondence between the performance in the paper table and the pretrained models, we find the pretrained models are hard to link to the reported paper results. Also, could you provide the model from DARTS searched, as we find that the searching cannot get same advantage over baseline model reported in the paper, we only get 3.05 test error.

**Comment by zhoudaquan** [comment]
> @zhoudaquan Hi thanks for sharing the work, as from #6 could you pls give a correspondence between the performance in the paper table and the pretrained models, we find the pretrained models are hard to link to the reported paper results. Also, could you provide the model from DARTS searched, as we find that the searching cannot get same advantage over baseline model reported in the paper, we only get 3.05 test error.

Can I check which model you found hard to link to the paper table? While I need some time to prepare the documentation for the readme, it is faster to directly give you the model you want now.

I will try to prepare the models from darts and pass to you later.

**Comment by linlililin123456** [comment]
thanks for the quick reply, I mean the naming, how to link to the table reported results? we are now trying to compare mobilenext with our model in detail, so need to reproduce all results.
![image](https://user-images.githubusercontent.com/73564467/97378636-c5d81f80-18fd-11eb-9eab-247e623a4283.png)
thanks for sharing the darts model, could you also share the searching code?

**Comment by zhoudaquan** [comment]
> thanks for the quick reply, I mean the naming, how to link to the table reported results? we are now trying to compare mobilenext with our model in detail, so need to reproduce all results.
> ![image](https://user-images.githubusercontent.com/73564467/97378636-c5d81f80-18fd-11eb-9eab-247e623a4283.png)
> thanks for sharing the darts model, could you also share the searching code?

Hi, Thanks for pointing out the naming issue. the 74.02 corresponding to the mobilenext 1.0 model and 74.02% is the val accuracy. The 76.05% is the mobilenext model with se module added. I have modify the naming just now.

The darts training code is cloned from the official repo released by the author.

**Comment by linlililin123456** [comment]
thanks! will evaluate the pretrained models and report here! for darts, have you made any modification? could you pls provide the searching log and searched model reported in the paper? @zhoudaquan

**Comment by zhoudaquan** [comment]
> thanks! will evaluate the pretrained models and report here! for darts, have you made any modification? could you pls provide the searching log and searched model reported in the paper? @zhoudaquan

Hi, sorry for this late reply. Will prepare it and send to you later this week.
